temporalcli/commandsgen/parse.go

```diff
@@ -19,15 +20,16 @@ var CommandsYAML []byte
 type (
 	// Option represents the structure of an option within option sets.
 	Option struct {
-		Name        string   `yaml:"name"`
-		Type        string   `yaml:"type"`
-		Description string   `yaml:"description"`
-		Short       string   `yaml:"short,omitempty"`
-		Default     string   `yaml:"default,omitempty"`
-		Env         string   `yaml:"env,omitempty"`
-		Required    bool     `yaml:"required,omitempty"`
-		Aliases     []string `yaml:"aliases,omitempty"`
-		EnumValues  []string `yaml:"enum-values,omitempty"`
+		Name         string   `yaml:"name"`
+		Type         string   `yaml:"type"`
+		Description  string   `yaml:"description"`
+		Short        string   `yaml:"short,omitempty"`
+		Default      string   `yaml:"default,omitempty"`
+		Env          string   `yaml:"env,omitempty"`
+		Required     bool     `yaml:"required,omitempty"`
+		Aliases      []string `yaml:"aliases,omitempty"`
+		EnumValues   []string `yaml:"enum-values,omitempty"`
+		Experimental bool     `yaml:"experimental,omitempty"`
```

Comment:
Is this used by any commands today? Want to set it on the YAML for the ones needed and/or wait until needed to add it?

Reply:
:+1: I see, so we had it in YAML but not in the parsed structure. I think this shows a gap in the strictness of our parser that you can add any YAML key. We should probably fix that, but doesn't need to be done here.